app/controllers/scenarios_controller.rb

```diff
@@ -95,6 +95,16 @@ def update
     end
   end
 
+  def enable_or_disable_all_agents
+    @scenario = current_user.scenarios.find(params[:id])
+    @scenario.agents.update_all disabled: params[:scenario][:disabled]
+
+    respond_to do |format|
+      format.html { redirect_to @scenario }
+      flash[:notice] = 'The agents has been successfully updated.'
+    end
+  end
+
```

Comment:
I wasn't sure if we are going to use rubocop from now on to check the code style. I actually grab the yml file from your pr. I will put what I had back in later.